OpenMEEG/cmake/UseAuto.cmake

```diff
@@ -20,8 +20,8 @@ if (USE_AUTO)
     if (WIN32)
         priorities(MKL OpenBLAS LAPACK)
     elseif (APPLE)
-        priorities(OpenBLAS vecLib LAPACK)
+        priorities(MKL OpenBLAS vecLib LAPACK)
     else()
-        priorities(OpenBLAS Atlas LAPACK)
+        priorities(MKL OpenBLAS Atlas LAPACK)
     endif()
 endif()
```

Comment:
You can easily merge the 3 cases. Define a ALTERNATE_MATH_LIB (or other good name) to empty if WIN32, to veclib if MACOS and to Atlas otherwise. Then you can have one single priorities line.